backend/app/usecase/url/urlcreator_test.go

```diff
@@ -24,6 +24,7 @@ func TestURLCreatorPersist_CreateURL(t *testing.T) {
 
 	alias := "220uFicCJj"
 	longAlias := "an-alias-cannot-be-used-to-specify-default-arguments"
+	emptyAlias := ""
```

Comment:
Let's hard code empty string in the test case so that reader can easily exam it works.